app/views/shared/_navbar.html.erb

```diff
@@ -0,0 +1,16 @@
+ <% if current_user %>
+<div class="w-full dark:text-white">
+  <div >
+    <nav style="background: #3778c2; min-height: 70px; display: flex; padding: 0 24px; align-items: center; justify-content: space-between;">
+          <div class="">
+            <%=link_to "<", 'javascript:history.back()', style: "color: white; text-decoration: none;font-weight: bolder;font-size: 8vw" %>
```

Comment:
- [**Optional**] I don't think this arrow is important on the home page(Categories). I think you should only use this arrow for navigation and not to log out of the app.

Reply:
- Addressed 💪🏻 🥇